<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePickerDialog</class>
 <widget class="QDialog" name="FilePickerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose file</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="MainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <widget class="Armed::ProjectTreeView" name="ProjectTree"/>
     <widget class="QFrame" name="PreviewArea">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File name:</string>
       </property>
       <property name="buddy">
        <cstring>FileName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FileName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Armed::ProjectTreeView</class>
   <extends>QTreeView</extends>
   <header>ProjectTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
